### PR TITLE
Feature/ATG-95 Leaderboard //best, current

### PR DESCRIPTION
**Pull request by IraCherginyshka**
New route /logout

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -57,8 +69,34 @@ app.route('/login').post((req, res) => {
...
+    })
+    .catch(() => errorHandler(res));
+});
+
+app.get('/result', (req, res) => {
+  Result.find({})
+    .sort({ score: -1 })
+    .then((sortResult) => {
+      res.send(sortResult);
+    });
```
Comment (oleksandrstarov) [inline server/app.js]: can we do this in one line?

File: src/app/components/game-leaderboard/game-leaderboard.component.html [hunk 2]
```diff
@@ -8,27 +8,29 @@
       <li class="leaderboard__field">lines</li>
     </ul>
   </header>
-  <ul class="leaderboard" *ngFor="let result of gameResults; let i = index">
-    <li class="leaderboard__list">
+  <ul class="leaderboard" *ngFor="let result of gameResults; let i = index" [class]="">
```
Comment (oleksandrstarov) [inline src/app/components/game-leaderboard/game-leaderboard.component.html]: why do we need to use an empty class?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 1 above)
Comment (IraCherginyshka) [reply server/app.js]: Yes
done

File: src/app/components/game-leaderboard/game-leaderboard.component.html (hunk 2 above)
Comment (IraCherginyshka) [reply src/app/components/game-leaderboard/game-leaderboard.component.html]: this is my mistake
Deleted

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**